pkg/kubelet/custommetrics/custom_metrics.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package custommetrics contains support for instrumenting cAdvisor to gather custom metrics from pods.
+package custommetrics
+
+import (
+	"path"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	CustomMetricsDefinitionContainerFile = "definition.json"
+
+	CustomMetricsDefinitionDir = "/etc/custom-metrics"
```

Comment:
I agree that cadvisor configuration should go in /etc/cadvisor. I don't think /etc/custom-metrics is a good place for this. It's overly generic.